websocket/frontend/src/todolist/Todoform.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+
+class TodoForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      api_url: props.api_url,
```

Comment:
1. Why do you need this (`api_url`) in the state?
2. Why snake case?

Reply:
the `api_url` is from the parent component (TodoList) and passed here as props so that I can access the same api_url.

Reply:
oh ya misunderstood the question. I will take it out from state